content/chainguard/libraries/python/build-configuration.md

```diff
@@ -286,6 +293,11 @@ poetry source add chainguard https://libraries.cgr.dev/python/simple/
 The [Poetry documentation](https://python-poetry.org/docs/) contains more
 information about your project build, dependencies, versions, and other aspects. 
 
+If you want to install Python libraries from multiple indexes with a priority 
+index resolution order, `poetry` supports setting a [primary package source](https://python-poetry.org/docs/repositories/#project-configuration). 
+For example, some organizations may choose to prioritize the Chainguard Python 
```

Comment:
I think we should be stronger and recommend that .. probably use the same wording as in the repo manager section. For many use cases it is essentially required